action.yml

```diff
@@ -0,0 +1,129 @@
+name: 'Build and push images to registry'
+description: 'This GitHub Action builds a container image and pushes it to the specified registry.'
+author: 'RHSCL team'
+branding:
+  icon: circle
+  color: blue
+
+inputs:
+  registry:
+    description: 'Registry to push container image to'
+    required: true
+  registry_namespace:
+    description: 'Namespace of the registry, where the image would be pushed'
+    required: true
+  registry_username:
+    description: 'Login to specified registry'
+    required: true
+  registry_token:
+    description: 'Token to access the specified registry'
+    required: true
+  version:
+    description: 'Version of the built image'
+    required: true
+  tag:
+    description: 'Tag of the built image'
+    required: true
+  dockerfile:
+    description: 'Dockerfile to build the image'
+    required: false
+    default: 'Dockerfile'
+  docker_context:
+    description: 'Docker build context'
+    required: false
+    default: '.'
+  use_distgen:
+    description: 'The action will use distgen for generating dockerfiles if true'
+    required: false
+    default: 'false'
+
```

Comment:
Output of this action is totally missing. Like references.

Reply:
Can you please explain more, what do you mean? I am not sure I understand.

Reply:
In case the user wants to use push references of build and push images, then they can use it. But this is not blocking.

Reply:
@zmiklank TODO: create issue for this